engine/apps/api/views/alert_group.py

```diff
@@ -85,7 +85,11 @@ class AlertGroupFilter(DateRangeFilterMixin, ModelFieldFilterMixin, filters.Filt
     invitees_are = filters.ModelMultipleChoiceFilter(
         queryset=get_user_queryset, to_field_name="public_primary_key", method="filter_invitees_are"
     )
+    involved_users_are = filters.ModelMultipleChoiceFilter(
```

Comment:
Probably we can name it just "Involved users" instead of "Involved users are"?

Reply:
I like that better too, but we already had "Invitees are". We may change both in a later PR?